grpc/errors_test.go

```diff
@@ -30,7 +30,7 @@ func TestErrorWrapping(t *testing.T) {
 	srv := grpc.NewServer(grpc.UnaryInterceptor(si.intercept))
 	es := &errorServer{}
 	testproto.RegisterChillerServer(srv, es)
-	lis, err := net.Listen("tcp", ":")
+	lis, err := net.Listen("tcp", "127.0.0.1:")
```

Comment:
Out of curiosity, why was this needed?

Reply:
Oh, I mean't to test if this was actually needed on Travis but forgot, with `:` my in-docker tests were timing out while trying to bind to an interface which was super weird.